### PR TITLE
Refactor 'categories' to be an enum type

### PR DESCRIPTION
**Pull request by tdaugaard**
As the title states. A simple refactor to eliminate literal strings from comparisons and assignments.

### CODE REVIEW

File: src/renderer/rendererutils.ts [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
+        VideoCategory.SoloShuffle,
+        VideoCategory.Battlegrounds,
     ];
 
-    if (pvpCategories.includes(category)) {
-        return true;
-    } else {
-        return false;
-    }
+    return pvpCategories.indexOf(category) !== -1;
```
Comment (aza547) [inline src/renderer/rendererutils.ts:17]: Why not just `.includes()`?

File: src/renderer/VideoButton.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ export default function VideoButton(props: any) {
   const videoIndex = props.index;
 
   const categoryIndex = state.categoryIndex;
-  const category = categories[categoryIndex]
+  const category = categories[categoryIndex] as VideoCategory
```
Comment (aza547) [inline src/renderer/VideoButton.tsx:24]: Semicolon

**Review by aza547 — CHANGES_REQUESTED** [review]
A few clean-up comments, otherwise looks good.

File: src/renderer/rendererutils.ts (hunk 1 above)
Comment (tdaugaard) [reply src/renderer/rendererutils.ts:17]: My bad. No idea why I thought that was a good idea to change.

**Pull request merged**